src/DeepLinkModule/javasource/deeplink/implementation/handler/DeeplinkHandler.java

```diff
@@ -63,7 +63,7 @@ protected void processRequest(IMxRuntimeRequest request, IMxRuntimeResponse resp
 				session = sessionFromRequest;
 			}
 
-			sessionContext = session.createContext();
+			sessionContext = session.createContext().createSudoClone();

```

Comment:
I think this change should be reverted. We don't need it to `getUser`, and for the pending link we already create a sudo clone below.